common_repo_templates/CONTRIBUTING_template.md

```diff
@@ -0,0 +1,51 @@
+**Table of Contents**
```

Comment:
still keep these common_repo_templates in community repo.

Reply:
removed as suggested.